test/gtest/miopendriver_regression_issue_1576.cpp

```diff
@@ -57,7 +56,7 @@ void RunMIOpenDriver()
 {
     using e_mask = enabled<Gpu::gfx94X, Gpu::gfx103X, Gpu::gfx110X>;
     using d_mask = disabled<Gpu::Default>;
-    if(!ShouldRunMIOpenDriverTest<d_mask, e_mask>("--half", true))
+    if(!ShouldRunMIOpenDriverTest<d_mask, e_mask>())
```

Comment:
I think this one should be swapped from Smoke to Full.

Similar to above, I think ShouldRunMIOpenDriverTest with a true argument meant to check MIOPEN_TEST_ALL, and I think that matches full tests now.

Reply:
Done